modules/nw-overlapped-sharding.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-overlapped-sharding_{context}"]
+== Overlapped sharding example
+
+In addition to `finops-router` and `dev-router` in the previous example, you also have `devops-router`, which is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev` and `ops`:
+
+.Example YAML definition for `devops-router`
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: devops-router
+  namespace: openshift-ingress-operator
+spec:
+  namespaceSelector:
+    matchLabels:
+      name:
+        - dev
+        - ops
```

Comment:
Better to change matchLabels with matchExpressions for matching multiple values
```
spec:
  namespaceSelector:
    matchExpressions:
    - key: name
      operator: In
      values:
      - dev
      - ops
```

Reply:
:heavy_check_mark: